charts/cluster-addons/values.yaml

```diff
@@ -367,6 +367,38 @@ nodeProblemDetector:
     namespace: node-problem-detector
     values: {}
 
+restrictedAdmission:
+  enabled: false
+  chart:
+    repo: https://stackhpc.github.io/restricted-pss-vaps-helm/
+    name: restricted-pss-vaps
+    version: 0.3.0
```

Comment:
we need to make sure we auto update this:
https://github.com/azimuth-cloud/capi-helm-charts/blob/dd78906a9939932ecff01ffe254b344a37b3a967/.github/workflows/update-addons.yml#L69